pytest_django/fixtures.py

```diff
@@ -229,16 +229,16 @@ def admin_user(db, django_user_model, django_username_field):
     try:
         user = UserModel._default_manager.get(**{username_field: 'admin'})
     except UserModel.DoesNotExist:
-        extra_fields = {}
-        if username_field != 'username':
-            extra_fields[username_field] = 'admin'
+        extra_fields = {username_field: 'admin'}
+        if username_field == 'email':
+            extra_fields[username_field] = 'admin@example.com'
```

Comment:
This could be a single statement / line.

    extra_fields[username_field] = 'admin@example.com' if username_field == 'email' else 'admin'